src/ttsim/plot/dag/tt.py

```diff
@@ -229,49 +275,53 @@ def _get_node_descriptions(
                 and n.description["en"] is not None
             ):
                 descr = n.description["en"]
-        if not descr:
-            descr = "No description available."
-        # Wrap description at 79 characters
-        descr = textwrap.fill(descr, width=79)
-        out[qn] = descr
+            else:
+                continue
+            # Wrap description at 79 characters
+            descr = textwrap.fill(descr, width=79)
+            out[p] = descr
     return out
 
 
-def _create_dag_with_selected_nodes(
-    complete_dag: nx.DiGraph,
-    node_selector: _QNameNodeSelector,
+def select_nodes_from_dag(
+    complete_tt_dag: nx.DiGraph,
+    tt_targets_qnames: set[str],
+    selection_type: Literal["neighbors", "descendants", "ancestors", "nodes"],
+    selection_depth: int | None = None,
 ) -> nx.DiGraph:
     """Select nodes based on the node selector."""
     selected_nodes: set[str] = set()
-    if node_selector.type == "nodes":
-        selected_nodes.update(node_selector.qnames)
-    elif node_selector.type == "ancestors":
-        for node in node_selector.qnames:
+    if selection_type == "nodes":
```

Comment:
Please use the same order as in the `Literal[]` definition, makes it easier to read.